docs/helpers/Eventline.md

```diff
@@ -29,6 +32,25 @@ When using Eventline with graph then `x axis type` must be **timeseries** it is
 | ------------- | -------- | -------------------------- | ------------------------------------------------- |
 | style         | object   | `{strokeDashArray: "2,2"}` | Any strokeDashArray value to add dash to the line |
 
+# Pass Through
+
+## JSON Properties
+
+#### Optional
+
+| Property Name    | Expected | Default | Description                                       |
+| ---------------- | -------- | ------- | ------------------------------------------------- |
+| clickPassThrough | object   | null    | Refer [Click Pass Throughs](#click-Pass-Throughs) |
```

Comment:
Might want to provide the link to dateline's click pass through instead of duplicating this.

Reply:
On second thought, there is no onClick so might want to change this accordingly.

Reply:
yes, I will do the change and I will not pass dateline click pass through link.